pkg/kubernetes/deployer.go

```diff
@@ -0,0 +1,76 @@
+package kubernetes
+
+import (
+	"fmt"
+	"time"
+
+	apps "k8s.io/api/apps/v1"
+	core "k8s.io/api/core/v1"
+)
+
+func (k *Kubernetes) GetDeployerResult(namespace string, pod *core.Pod) (string, string) {
```

Comment:
Okay you can change this function name to something like `GetFileFromPod` adding `path` argument. Then, It can be moved to `controller.go` as I say in today's call. Thank you! 🙏

Reply:
And please check below functions when you are okay. 😍

Reply:
@rlgns98kr
I am sorry, I missed it. Could you explain again?

Reply:
I updated!